metrics-reporter-hive/src/main/java/org/apache/kylin/metrics/lib/impl/hive/HiveProducer.java

```diff
@@ -97,7 +105,10 @@ public void onRemoval(RemovalNotification<Pair<String, String>, Pair<String, Lis
         } catch (UnknownHostException e) {
             hostName = "UNKNOWN";
         }
-        CONTENT_FILE_NAME = hostName + "-part-0000";
+        CONTENT_FILE_NAME = hostName + "-part-";
+        String fsUri = fileSystem.getUri().toString();
+        supportAppend = !fsUri.startsWith("s3") && !fsUri.startsWith("wasb"); // AWS EMR and Azure HDInsight
```

Comment:
may need to cover more cloud storage:
- adls (azure data lake)
- gs (google storage)
- oss (aliyun oss) 

or, can we just treat "hdfs" as the append case, all others as non-append?